src/pages/iou/request/step/IOURequestStepConfirmation.js

```diff
@@ -228,6 +229,13 @@ function IOURequestStepConfirmation({
         (selectedParticipants) => {
             const trimmedComment = lodashGet(transaction, 'comment.comment', '').trim();
 
+            // Don't let the form be submitted multiple times while the navigator is waiting to take the user to a different page
+            if (formHasBeenSubmitted) {
```

Comment:
Shouldn't we be disabling the button?

Reply:
The button is about 5-6 nested components deep and would need a new prop. This problem isn't unique to this form (but getting the user's location doesn't help since it adds a little delay). so I didn't feel that adding a new disabled prop, and drilling it down, just for this form would be very valuable.